.github/workflows/deploy-dev.yml

```diff
@@ -26,10 +26,10 @@ jobs:
     needs: build
     with:
       deploy_to_environment: dev
-      lambda: dev-ep-event-delivery
+      lambda: dev-ep-indexer
       docker_image_version: ${{ needs.build.outputs.build_date }}-dev
     secrets:
-      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
+      aws_region: us-west-2
```

Comment:
should we use variable?

Reply:
https://github.com/elastic-ipfs/event-delivery-lambda/pull/1#discussion_r930239640